API/index.js

```diff
@@ -0,0 +1,59 @@
+let input = document.querySelector("input");
+let imgContainer = document.querySelector(".photo")
+let inputValue;
+
+function imgCreate(data) {
```

Comment:
how do we name funcs? it should start with a verb

Reply:
fixed